src/plugins/intel_cpu/tests/unit/transformations/x64/convert_matmul_test_3d_weight.cpp

```diff
@@ -0,0 +1,106 @@
+// Copyright (C) 2018-2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include <memory>
+#include <openvino/core/model.hpp>
```

Comment:
```suggestion
#include "openvino/core/model.hpp"
```
Use `""` for OV includes

Reply:
Done.